var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -54,16 +56,14 @@ def url_for_version(self, version):
         else:
             return "{0}/v{1}.tar.gz".format(base_url, version)
 
-    def install(self, spec, prefix):
-        # Generate configure
+    def autoreconf(self, spec, prefix):
         libtoolize()
         aclocal('-I', 'lib/autoconf')
         autoconf()
 
-        config_args = [
-            '--prefix={0}'.format(prefix),
-            '--enable-shared'
-        ]
+    def configure_args(self):
+
+        cargs = ['--enable-shared']
```

Comment:
Calling this `config_args` would both be clearer, and result in fewer changes to the code.